src/runtime/storageNG/abstract-store.ts

```diff
@@ -174,3 +177,11 @@ export type StoreInfo = {
    */
   readonly includeKey?: string;
 };
+
+/**
+ * Dataflow claims defined on a store. Maps from target field to a list of tags
+ * claimed on the field. Target can be an empty string, meaning it applies to
+ * the entire schema for the store, or a dotted string pointing to a field
+ * inside it (e.g. someField.myRef.foo).
+ */
+export type StoreClaims = Map<string, ClaimIsTag[]>;
```

Comment:
Won't we need to support other claims in addition to `ClaimIsTag`?

Reply:
There's only two types of claims so far: ClaimIsTag and ClaimDerivesFrom. Do you think we'll want to support stores claiming they derive from somewhere else?

We can come back and update this later, but for now I've just preserved the existing behaviour which is that they can only make ClaimIsTag claims.